mlos_vizpanel/backend.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime, timedelta
```

Comment:
Module docstring and/or some README.md with details on what this is 
a) supposed to do,
b) how it does it,